### ISSUE TITLE
cabal install fails with GHC 7.8.4 on Linux with amd64

### ISSUE THREAD

**Issue by josyoun** [nikita-volkov/record#6]
I'm getting a cabal installation fail with:

```
Building record-0.1.4...
Failed to install record-0.1.4
Build log ( /home/josyoun/.cabal/logs/record-0.1.4.log ):
[1 of 1] Compiling Main             ( /tmp/record-0.1.4-19517/record-0.1.4/dist/setup/setup.hs, /tmp/record-0.1.4-19517/record-0.1.4/dist/setup/Main.o )
Linking /tmp/record-0.1.4-19517/record-0.1.4/dist/setup/setup ...
Configuring record-0.1.4...
Building record-0.1.4...
Preprocessing library record-0.1.4...
[1 of 4] Compiling Record.Parser    ( library/Record/Parser.hs, dist/build/Record/Parser.o )

library/Record/Parser.hs:283:7:
    Not in scope: `match'
    Perhaps you meant `catch' (imported from BasePrelude)
cabal: Error: some packages failed to install:
record-0.1.4 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by nikita-volkov** [comment]
Oh, yes. It appears, this function only appeared in "attoparsec" 0.12, while "record" specifies in its dependencies that it supports earlier versions of that library. Hence for the time being you can resolve this by manually forcing to choose an appropriate version:

```
cabal install record --constraint=attoparsec==0.12.*
```

**Comment by nikita-volkov** [comment]
Fixed in 0.1.5
